src/lib/internationalization/locales/zh.cts

```diff
@@ -408,12 +447,78 @@ export = buildIncompleteTranslation({
     theme_search: "搜索",
     theme_menu: "菜单",
     theme_permalink: "永久链接",
-    tag_see: "参阅",
-    tag_group: "所属分组",
-    tag_example: "示例",
     theme_copy: "复制",
     theme_copied: "已复制！",
     theme_normally_hidden: "由于您的过滤器设置，该成员已被隐藏。",
     theme_class_hierarchy_title: "类继承图表",
     theme_loading: "加载中……",
+
+    tag_defaultValue: "默认值",
+    tag_deprecated: "已被弃用",
+    tag_example: "示例",
+    tag_param: "参数",
+    tag_privateRemarks: "私有备注",
+    tag_remarks: "备注",
+    tag_returns: "返回",
+    tag_see: "参阅",
+    tag_throws: "抛出",
+    tag_typeParam: "类型参数",
+
+    tag_author: "作者",
+    tag_callback: "回调",
+    tag_category: "类别",
+    tag_categoryDescription: "类别描述",
+    tag_default: "默认值",
+    tag_document: "文档",
+    tag_extends: "继承自",
+    tag_augments: "继承自",
+    tag_yields: "生成",
+    tag_group: "分组",
+    tag_groupDescription: "分组描述",
+    tag_import: "导入",
+    tag_inheritDoc: "继承文档",
+    tag_jsx: "JSX",
+    tag_license: "许可协议",
+    tag_module: "模块",
+    tag_prop: "属性",
+    tag_property: "属性",
+    tag_return: "返回",
+    tag_satisfies: "满足",
+    tag_since: "添加于",
+    tag_template: "类型参数",
+    tag_type: "类型",
+    tag_typedef: "类型定义",
+    tag_link: "链接",
+    tag_label: "标记",
+
+    tag_linkcode: "链接",
+    tag_linkplain: "链接",
+
+    tag_alpha: "alpha",
+    tag_beta: "beta",
+    tag_eventProperty: "事件属性",
+    tag_experimental: "实验性",
+    tag_internal: "内部成员",
+    tag_override: "重写",
+    tag_packageDocumentation: "包文档",
+    tag_public: "公共成员",
+    tag_readonly: "只读",
+    tag_sealed: "无法继承",
+    tag_virtual: "虚函数",
+    tag_abstract: "抽象类",
+    tag_class: "类",
+    tag_enum: "枚举",
+    tag_event: "事件",
+    tag_hidden: "隐藏",
+    tag_hideCategories: "在类别中隐藏",
```

Comment:
It doesn't hurt anything to have these here, but modifier tags that typedoc applies behavior to, like this one, won't ever be rendered.